client/my-sites/site-settings/disconnect-site/index.jsx

```diff
@@ -0,0 +1,52 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { flowRight } from 'lodash';
+import { localize } from 'i18n-calypso';
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import DocumentHead from 'components/data/document-head';
+import FormattedHeader from 'components/formatted-header';
+import { getSelectedSite } from 'state/ui/selectors';
+import Main from 'components/main';
+import PaginationFlow from './pagination-flow';
+import Placeholder from 'my-sites/site-settings/placeholder';
+import redirectNonJetpackToGeneral from 'my-sites/site-settings/redirect-to-general';
+
+class DisconnectSite extends Component {
+	render() {
+		const { site, translate } = this.props;
+
+		if ( ! site ) {
+			return <Placeholder />;
+		}
+		return (
+			<Main className="disconnect-site site-settings">
+				<DocumentHead title={ translate( 'Site Settings' ) } />
+				<FormattedHeader
+					headerText={ translate( 'Disconnect Site' ) }
+					subHeaderText={ translate(
+						'Tell us why you want to disconnect your site from Wordpress.com.'
+					) }
+				/>
+				<Card className="disconnect-site__card"> </Card>
+				<PaginationFlow />
+			</Main>
+		);
+	}
+}
+
+const connectComponent = connect( state => {
+	return {
+		site: getSelectedSite( state ),
+	};
+} );
+
+export default flowRight( connectComponent, localize, redirectNonJetpackToGeneral )(
```

Comment:
Nit: I prefer listing each of those on separate lines, like you've done it for `redirectNonJetpackToGeneral`. This is also valid though, so it's your call.

Reply:
@tyxla, I did also here and when I submit the code the lines get merged. The same story happens in other cases at the very end of the files. I suspect that my editor decided to go independent 😂 . 
Seriously speaking though, I will fix it 👍